apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -379,7 +379,7 @@ public int MaxConcurrentStreamsLowWatermark
 
         /// <summary>
         /// Defines the default values for <see cref="SpannerCommand.CommandTimeout"/> and
-        /// <see cref="SpannerTransaction.CommitTimeout"/> along with all network operations to a Cloud
+        /// <see cref="SpannerTransactionOptions.CommitTimeout"/> along with all network operations to a Cloud
```

Comment:
Hmm... this is interesting, because if someone fetches the SpannerTransactionOptions.CommitTimeout somehow, they'll see null, right? It's more that it defines the default *effective* value for commit timeouts?

If I've understood that correctly, I'm happy to try to come up with some slightly weaker words.